SingularityBase/src/main/java/com/hubspot/singularity/SingularityDeployProgress.java

```diff
@@ -56,21 +66,22 @@ public long getTimestamp() {
   }
 
   public SingularityDeployProgress withNewInstances(int instances) {
-    return new SingularityDeployProgress(instances, deployInstanceCountPerStep, deployStepWaitTimeMs, false, autoAdvanceDeploySteps, failedDeployTasks, System.currentTimeMillis());
+    return new SingularityDeployProgress(instances, currentActiveInstances, deployInstanceCountPerStep, deployStepWaitTimeMs, false, autoAdvanceDeploySteps, failedDeployTasks, System.currentTimeMillis());
   }
 
   public SingularityDeployProgress withCompletedStep() {
-    return new SingularityDeployProgress(targetActiveInstances, deployInstanceCountPerStep, deployStepWaitTimeMs, true, autoAdvanceDeploySteps, failedDeployTasks, System.currentTimeMillis());
+    return new SingularityDeployProgress(targetActiveInstances, currentActiveInstances + 1, deployInstanceCountPerStep, deployStepWaitTimeMs, true, autoAdvanceDeploySteps, failedDeployTasks, System.currentTimeMillis());
```

Comment:
is `currentActiveInstances + 1` correct here? wouldn't it be `currentActiveInstances + deployInstanceCountPerStep`?